app/views/groups/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="myclass">
+  <%= link_to "Back", groups_path, class:"back-link" %>
+  <p>DETAILS</p>
+  <div >
+  <%= link_to "Log Out", destroy_user_session_path,  class: "c-actions" %> 
+  </div>
+</div>
+<p style="color: green"><%= notice %></p>
+
+<% if @group.icon.attached? %>
+    <image class="icon-image" src="<%=(url_for(@group.icon)) %>>
+<% end %>
+
+<%= render @group %>
+
+
+<% @group.transactions.each do |transaction| %>
+  <div class="user-container-index">
+    <div class="user-info">
+      <p class="user-numbers"><strong>Transaction Name:</strong> <%= transaction.name %></p>
+      <p class="user-numbers"><strong>Amount:</strong> $<%= transaction.amount %></p>
+      <p class="user-numbers"><strong>Date:</strong> <%= transaction.created_at %></p>
+      <hr>
+    </div>
+</div>
+<% end %>
+<div>
+<% myamt = 0 %>
+<% @group.transactions.each do |transaction| %>
+  <% myamt += transaction.amount %>
+<% end %>
+<d class="payment">
+<p class="payment1"><strong>Total Payment:</strong></p> 
+<p class="payment2">$<%= myamt  %></p>
+</div>
+
+<%= link_to "Add a new Transaction", '/groups/:group_id/transactions/new', class:"btn btn-light my-cstm-btn" %>
+
```

Comment:
- [ ] There is a problem with your routing in this file where you have a link to "Add a new transaction". You can use route names in this case `new_group_transaction_path(@group)`. That should fix your route 😎.You can see route error below 👇 
![image](https://user-images.githubusercontent.com/46839548/176786326-c1cce986-c8ed-4fe0-9390-ff39ee8f9eea.png)